packages/suite/src/views/wallet/send/components/Options/BitcoinOptions/BitcoinOptions.tsx

```diff
@@ -49,6 +49,10 @@ const StyledButton = styled(Button)`
     margin: 4px 8px 4px 0;
 `;
 
+const Inline = styled.span`
+    display: inline-flex;
```

Comment:
why not simply style the `Button`? 👀

Reply:
button needs to be `display: block` to make overflow: ellipsis work

Reply:
The filosofy here to make interface of component simple, if you need custom layouting of the inner components do it seperately

Reply:
Not the button itself but the `Content` within. Just an **nit** idea for a clearer style with less code :) 

```
> div:last-child {
    display: inline-flex;
}
```

Reply:
Ah got it, instead of changing Button compont as a while you just suggest to use css magic instead of adding ann extra layer of components. 

I see, this is a tradeoff I am not sure about. @jvaclavik WDYT?

Reply:
My opinion (not sure yetú is, that you shall not re-style the inner parts of the component from outside via CSS. This brakes the open-close principle.